examples/e2e/app-pages-router/e2e/headers.test.ts

```diff
@@ -0,0 +1,20 @@
+import { expect, test } from "@playwright/test";
+
+test("should test if poweredByHeader adds the correct headers ", async ({ page }) => {
+	const result = await page.goto("/ssg");
+	expect(result).toBeDefined();
+	expect(result?.status()).toBe(200);
+	const headers = result?.headers();
+
+	// This header should not be defined even when cache interception happens in the external middleware
+	expect(headers?.["x-opennext-requestid"]).toBeFalsy();
```

Comment:
Would the following be more in line with what we want to test?

```suggestion
	expect(headers?.["x-opennext-requestid"]).toBeUndefined();
```